pose-detection/demo/src/index.js

```diff
@@ -93,11 +91,21 @@ async function renderPrediction() {
 
 async function app() {
   await tf.setBackend('webgl');
-  setupDatGui();
+
+  // Gui content will change depending on which model is in the query string.
+  const urlParams = new URLSearchParams(window.location.search);
+  if (!urlParams.has('model')) {
```

Comment:
Are you sure you don't want to use a default when there's no URL parameter for the model type?